staging/src/k8s.io/apimachinery/pkg/api/apitesting/naming/naming.go

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
wouldn't pkg/apitesting or even pkg/api/testing fit better?

Reply:
This directory already exists. Don't ask me why it's named this way, but I think this place would make the most sense vs creating a new dir.